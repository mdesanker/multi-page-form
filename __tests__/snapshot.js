import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Home from "../pages/index";

it("renders homepage unchanged", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
