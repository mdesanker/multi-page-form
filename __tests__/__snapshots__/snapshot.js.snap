// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="css-umsz9e"
  >
    <button
      class="chakra-button css-5c2176"
      href="/form"
      type="button"
    >
      Create a Bounty
    </button>
  </div>
</div>
`;
